src/Microsoft.OpenApi.Readers/OpenApiReaderSettings.cs

```diff
@@ -61,11 +61,17 @@ public class OpenApiReaderSettings
         public Uri BaseUrl { get; set; }
 
         /// <summary>
-        /// Function used to provide an alternative loader for accessing external references.  
+        /// Function used to provide an alternative loader for accessing external references.
         /// </summary>
         /// <remarks>
         /// Default loader will attempt to dereference http(s) urls and file urls.
         /// </remarks>
         public IStreamLoader CustomExternalLoader { get; set; }
+
+        /// <summary>
+        /// Whether to leave the <see cref="Stream"/> object open after reading
+        /// from an <see cref="OpenApiStreamReader"/> object.
+        /// </summary>
+        public bool LeaveStreamOpen { get; set; } = false;
```

Comment:
```suggestion
        public bool KeepStreamOpen { get; set; };
```
boolean is always false by default.
nit on naming

Reply:
👍🏼 on the boolean default.

On the naming: I wanted to remain as close to the framework naming as possible: See: https://docs.microsoft.com/en-us/dotnet/api/system.io.streamreader.-ctor?view=netcore-3.1#System_IO_StreamReader__ctor_System_IO_Stream_System_Text_Encoding_System_Boolean_System_Int32_System_Boolean_
![image](https://user-images.githubusercontent.com/40403681/121017208-b24f3a00-c7a5-11eb-92ca-24c8ab147102.png)

Reply:
perfect, ignore my naming suggestion then. Thanks for doing the research